A32NX/html_ui/Pages/A32NX_Utils/SimBriefApi.js

```diff
@@ -1,10 +1,15 @@
 class SimBriefApi {
-    static getSimBriefOfp(username) {
-        if (!username) {
-            throw ("No SimBrief username provided");
+    static getSimBriefOfp(username, userId) {
+        var apiUrl;
```

Comment:
Use let instead of var for code consistency

Reply:
Non-JS dev figuring this out as I go along 😛 Thanks, fixed.

Reply:
JS is confusing even when declaring a simple variable 🤣